src/api/app/controllers/source_controller.rb

```diff
@@ -544,8 +544,8 @@ def update_project_meta
       else
         project = Project.new(name: project_name)
         project.update_from_xml!(request_data)
-        # failure is ok
-        project.add_user(User.current.login, 'maintainer')
+        # FIXME3.0: don't modify send data
+        project.relationships.build(user: User.current, role: Role.find_by_title!('maintainer'))
```

Comment:
On Freitag, 23. Oktober 2015, 06:13:11 CEST wrote Henne Vogelsang:
> > @@ -544,8 +544,8 @@ def update_project_meta
> >        else
> >          project = Project.new(name: project_name)
> >          project.update_from_xml!(request_data)
> > -        # failure is ok
> > -        project.add_user(User.current.login, 'maintainer')
> > +        # FIXME3.0: don't modify send data
> > +        project.relationships.build(user: User.current, role: Role.find_by_title!('maintainer'))
> 
> So if I update the meta file I automatically become maintainer?

no, this is the case for creating a new project only.